includes/class-wc-stripe-intent-controller.php

```
@@ -715,7 +716,27 @@ public function update_order_status_ajax() {
 			$save_payment_method = isset( $_POST['payment_method_id'] ) && ! empty( wc_clean( wp_unslash( $_POST['payment_method_id'] ) ) );
 
 			$gateway = $this->get_upe_gateway();
+
+			$is_already_locked = $order_helper->lock_order_payment( $order );
+			if ( $is_already_locked ) {
+				WC_Stripe_Logger::debug(
+					'Skipped updating order status in ajax request because order is already being processed.',
+					[ 'order_id' => $order_id ]
+				);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Logger' in method 'update_order_status_ajax'. (undefined)

(StaticAccess)